index.js

```diff
@@ -1,6 +1,41 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
-  name: 'ember-redux'
+  name: 'ember-redux',
+
+  included: function(app) {
+    // Import UMD redux and redux-thunk
+    // transform to named AMD that we can import
+    app.import('vendor/redux.js', {
+      using: [{ transformation: 'amd', as: 'redux' }]
+    });
+    app.import('vendor/redux-thunk.js', {
+      using: [{ transformation: 'amd', as: 'redux-thunk' }]
+    });
+    // If developing the addon itself, add redux-saga
+    if (this.parent.pkg.name === 'ember-redux') {
+      app.import('vendor/redux-saga.js', {
+        using: [{ transformation: 'amd', as: 'redux-saga' }]
+      });
+    }
+  },
+
+  treeForVendor: function() {
+    // Grab the UMD distributions of redux and redux-thunk
+    var trees = [
+      path.join(path.dirname(require.resolve('redux')), '..', 'dist'),
+      path.join(path.dirname(require.resolve('redux-thunk')), '..', 'dist'),
+    ];
+    // If developing the addon itself, add redux-saga
+    if (this.parent.pkg.name === 'ember-redux') {
+      trees.push(
+        path.join(path.dirname(require.resolve('redux-saga')), '..', 'dist')
+      );
+    }
+    return mergeTrees(trees);
```

Comment:
The only problem there is that someone 4 months from now, will try to app.import something that would normally be there and it will be hard to track this down and realize that we're dropping the default vendorTree here. My preference would be to maintain the standard vendorTree even if it isn't used right now.